README.md

```diff
@@ -1,148 +1,153 @@
-# A Kubernates Cloud-Shell (Web Terminal) Operator.
+# A Kubernates Cloud-Shell (Web Terminal) Operator
+
 [![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/6301/badge)](https://bestpractices.coreinfrastructure.org/projects/6301)
 
 English | [简体中文](https://github.com/cloudtty/cloudtty/blob/main/README_zh.md)
 
-# Why needs cloudtty ?
+cloudtty is a kind of easy-to-use web terminal or cloud shell intended for a kubernetes-native environment. You can easily run the web tty as a pod on your own browser. The community is always open for any contributors and those who want to have a try.
```

Comment:
`run the web tty as a pod on your own browser` is not accurate ...

Reply:
How about this:
```
use the webtty as a pod on your own browser.
```
Not sure any more better wordings when I'm still in a very beginning understanding -_!

Reply:
you can easily open a terminal on your own browser via cloudtty?